interfaces/builtin/wayland.go

```diff
@@ -59,6 +59,7 @@ owner /run/user/[0-9]*/wayland-[0-9]* rwk,
 # Allow reading an Xwayland Xauth file
 # (see https://gitlab.gnome.org/GNOME/mutter/merge_requests/626)
 /run/user/[0-9]*/.mutter-Xwaylandauth.* r,
+/run/user/[0-9]*/mutter/Xauthority r,
```

Comment:
Looks like the path was introduced 2 months ago:
https://gitlab.gnome.org/GNOME/mutter/commit/a8984a81c2e887623d69ec9989ae8a5025f7bd47

Reply:
Duh, different path.